packages/gp2-components/src/molecules/EventOwner.tsx

```diff
@@ -0,0 +1,56 @@
+import { gp2 as gp2Model } from '@asap-hub/model';
+import { Link, neutral900, pixels } from '@asap-hub/react-components';
+import { gp2 as gp2Routing } from '@asap-hub/routing';
+import { css } from '@emotion/react';
+import { projectIcon, workingGroupIcon } from '../icons';
+import IconWithLabel from './IconWithLabel';
+
+const { rem } = pixels;
+
+const { projects, workingGroups } = gp2Routing;
+const listItemStyles = css({
+  padding: `${rem(7.5)} 0`,
```

Comment:
I think the IconWithLabel has this margin, we are removing it then adding it back here has a padding

Reply:
done